lldb/source/API/SBBreakpoint.cpp

```diff
@@ -342,7 +342,7 @@ uint32_t SBBreakpoint::GetIgnoreCount() const {
   return count;
 }
 
-void SBBreakpoint::SetThreadID(tid_t tid) {
+void SBBreakpoint::SetThreadID(lldb::tid_t tid) {
```

Comment:
If these lldb:: are fixing compilation errors they would be welcome as a separate patch.